03week/towersOfHanoi.js

```diff
@@ -19,23 +24,67 @@ function printStacks() {
   console.log("c: " + stacks.c);
 }
 
-function movePiece() {
-  // Your code here
+function movePiece(startStack, endStack) {
+     if(startStack === 'a' && (endStack === 'b' || endStack === 'c')) {
+       return true;
+     } else if (startStack === 'b' && (endStack === 'a' || endStack === 'c')) {
+       return true;
+     } else if (startStack === 'c' && (endStack === 'a' || endStack === 'b')) {
+       return true;
+     }
+}
 
+const isValid = (startStack, endStack) => {
+  try {
```

Comment:
this can be greatly simplified, and you don't need to use try/catch or throw, you're not calling a promise.